addons/l10n_ca/data/res_company_data.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo noupdate="1">
-<record model="res.company" id="base.main_company">
-    <field name="paperformat_id" ref="base.paperformat_us"/>
-</record>
+    <record model="res.company" id="base.main_company">
```

Comment:
Now we have a `demo_company_ca` company, this file is useless I think.
EDIT: Changing a demo data outside the `demo` folder seems to be a very bad idea...